xcodeproj/internal/info_plists.bzl

```diff
@@ -31,6 +31,12 @@ plutil -remove UIDeviceFamily \"{output}\" > /dev/null 2>&1 || true
 plutil -insert CFBundleIconFile -string \"{app_icon_path}\" \"{output}\" > /dev/null 2>&1 || true
 """.format(app_icon_path = default_app_icon_path, output = output.path)
 
+    if alternate_icon_names and len(alternate_icon_names):
+        for alternate_icon_name in alternate_icon_names:
+            command += """
+    plutil -insert CFBundleIcons.CFBundleAlternateIcons.\"{alternate_icon_name}\".CFBundleIconFiles -string \"{alternate_icon_name}\" > /dev/null 2>&1 || true
```

Comment:
@brentleyjones Is this the right way to do this? I am not seeing fixtures update and I know that we are getting here in the logic. When is this executed?

Reply:
No. The above stuff was only to make app icons appear on the schemes. We don't need to modify the plist, as the bazel generated one will have it in it alread.

Reply:
The fixture is being updated, it's just that the alternate icons are in the same `.xcasset` bundle. You should put it in another bundle to make it more visible that it is being parsed and picked up.